src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");
+    }
+
+    public void stopWork() {
```

Comment:
The `stopWork` method should be annotated with `@Override` to indicate that it is overriding a method from the base class `Machine`.